rnaseq-cgl-pipeline/test.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python2.7
+# John Vivian
+import subprocess
+import unittest
+
+
+class TestRNASeqPipeline(unittest.TestCase):
+
+    def test_docker_call(self):
+        out, err = check_docker_output(tool='quay.io/ucsc_cgl/rnaseq-cgl-pipeline')
```

Comment:
More robust tests are a bit difficult to develop as Mac OSX doesn't have a /var/run/docker.sock mount point, and I get an error from the wrapper when it tries to scrape it's internal docker container state. I'll bring it up at scrum.